libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -444,6 +444,33 @@ class Strings {
         assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
     }
 
+    @Sample
+    fun lastIndexOf() {
+        fun matchDetails(inputString: String, whatToFind: String, startIndex: Int = 0): String {
+            val matchIndex = inputString.lastIndexOf(whatToFind, startIndex)
+            return "Searching for '$whatToFind' in '$inputString' starting at position $startIndex: " +
+                    if (matchIndex >= 0) "Found at $matchIndex" else "Not found"
+        }
+
+        val inputString = "Never ever give up"
+        val toFind = "ever"
+
+        assertPrints(matchDetails(inputString, toFind), "Searching for 'ever' in 'Never ever give up' starting at position 0: Found at 6")
+        assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
```

Comment:
It is a good idea to add an example for the scenario when a substring could not be found at a specified index. However, for `startIndex = 10` the result will be different and the sample execution will fail. You can check it by executing `Strings.lastIndexOf` as a regular unit test.